src/components/NewEnvironment/index.js

```diff
@@ -124,6 +124,44 @@ const NewEnvironment = ({
                                 onBlur={() => toggleShowEnvType()}
                               />
                           </div>
+                          { environmentCount > 0 &&
+                          <div className="showHideContainer">
+                            <div className="copy-field deploy-key">
+                              <div className="field">
+                                <label>Deploy Key: </label>
+                                {loadingDK ? <div className="loader"></div>
+                                    : !showDKField ?
+                                        hashValue(dkValue).substring(0, 25)
+                                        : dkValue.length > 65 ?
+                                            dkValue.substring(0, 65) + '...'
+                                            : dkValue
+                                }
```

Comment:
While this works, not using nested ternaries improves readability. Maybe something like this could work: 


```suggestion
const renderValue = () => {
    if (loadingDK) {
        return < div className = "loader" > < /div>;
    }

    if (!showDKField) {
        return hashValue(dkValue).substring(0, 25);
    }

    return dkValue.length > 65 ? dkValue.substring(0, 65) + '...' : dkValue;
}

  <label>Deploy Key: </label>
  { renderValue()}

```